shared/macros-oval.jinja

```diff
@@ -0,0 +1,149 @@
+{{#
+  High level macro which checks for a particular set of parameter plus value in a configuration file.
```

Comment:
set of parameter**s**

Reply:
actually I'll change to `checks if a particular combination of parameter and value in a configuration file is set`

Reply:
Isn't plural still correct phrase for this , `combination of parameters`?

Reply:
But it is referring to combination which is single so `a combination of parameters is...`. IMO